### ISSUE TITLE
::selection makes --text-muted barely visible

### ISSUE THREAD

**Issue by jonaskuske** [kognise/water.css#203]
Both the `::selection` color and `--text-muted` are a sort-of similar shade of gray, so selecting text that's muted, e.g. a `<footer>` / `<cite>` means the text becomes very hard to read.

**Comment by kylejrp** [comment]
Not so great on the light theme: (Color contrast ratio of 1.69)
![image](https://user-images.githubusercontent.com/6742479/83483312-30705680-a457-11ea-8f45-f4ab1527b8d9.png)

Better, but still not great on the dark theme: (Color contrast ratio of 2.18)
![image](https://user-images.githubusercontent.com/6742479/83483279-2189a400-a457-11ea-84c9-e632899f66cc.png)

To fix, we would need to update the `--selection` color found in `/src/variables-light.css` so that it has a better color contrast ratio. Alternatively, we could keep the same background color but make the text darker when selected, which would require modifying the `selection` rules in `/src/parts/misc.scss`

**Comment by jonaskuske** [comment]
We could also use a different selection color by changing the selection color everywhere we use `--text-muted`. Something like this:

```css
color: var(--text-muted);
--selection: var(--selection-muted);
```

**Comment by habeckmatt** [comment]
Is this bug up for grabs?

**Comment by kognise** [comment]
@habeckmatt absolutely! If you wouldn't mind, can you edit `two` branch? We can merge this alongside #194

**Comment by kylejrp** [comment]
@habeckmatt And don't forget to ask for any help if you need it, especially if you're having trouble navigating or building the repository. We're here to help!

**Comment by habeckmatt** [comment]
So in this instance, would just changing the --selection color in variables-dark.cs and variables-light.css? Or should I go with what jonaskuske suggested and changed every spot where --text-muted is located?

**Comment by kognise** [comment]
Maybe consider adding a line to the styles of `::selection` to override the color?

**Comment by habeckmatt** [comment]
I also have another question about how to clone the two branch and make changes there as kognise suggested. I was able to fork and clone the repo locally, but I don't see a branch 'two'.

**Comment by kognise** [comment]
That's weird - what's the output of your `git branch` command?

**Comment by habeckmatt** [comment]
I think I was able to resolve that issue. I'm not seeing the *two branch showing.

**Comment by kognise** [comment]
Np :) 

Feel free to ask if you have any other questions, that's why we're here!

**Comment by habeckmatt** [comment]
So far I've just added: 'color: var(--text-bright);' to ::selection. I double checked on webaim.org and the contrast ratios were 4.71 for dark and 7.83 for light. Not sure if this is a wise stylistic choice or if I should change the '--selection:' color?

**Comment by kognise** [comment]
Mind making a PR and including screenshots? I can mark it as WIP and we can work from there 🤗

**Comment by habeckmatt** [comment]
Ok. So far I committed the changes to the 'two' branch in my forked repository, and the changes appear there. How do I make a pull request to your 'two' branch?

**Comment by kylejrp** [comment]
@habeckmatt You'll want to make sure that you've pushed your commits to your fork on GitHub, right now I think that they're just local changes on your computer. Once you've pushed, you should be able to [follow these steps](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) to make a pull request with a base of `kognise/water.css two` and a head of `habeckmatt/water.css two`.

**Comment by habeckmatt** [comment]
Perfect, I think I got it.

**Comment by kylejrp** [comment]
Closed by #210
